app-typescript/components/DatePicker.tsx

```diff
@@ -33,6 +33,8 @@ interface IDatePickerBase extends IInputWrapper {
         }}
      */
     locale?: DatePickerLocaleSettings;
+
+    // Hide does not mean show
```

Comment:
light does not mean dark ;D You'd need to phrase clearer. Or drop it completelly which would be fine here too.